autogen/agentchat/contrib/agent_eval/criterion.py

```diff
@@ -0,0 +1,69 @@
+import json
+from typing import List
+
+
+class Criterion:
```

Comment:
What do you think about using pydantic models for `Criterion`? This should simplify the logic for serializing and deserializing strings.

Reply:
sounds good, will make the change